### ISSUE TITLE
Jobs is not updated

### ISSUE THREAD

**Issue by Mahmoudgalalz** [zjunior-infra/Dashboard#18]
when updating the job, isn't get updated both in FE, and BE
which is database

**Comment by EL-Gazoly** [comment]
![Screenshot from 2023-05-03 16-24-03](https://user-images.githubusercontent.com/62439402/235929137-e2a5c8ae-a15d-418e-ae71-2b77c58a97af.png)
It works normally. I just added a company title for a couple of jobs, like Ziw and Veleo, and it was updated normally.

**Comment by Mahmoudgalalz** [comment]
@EL-Gazoly this issue has a follow back

the FE isn't getting updated I have to refresh the page, 

and also I can only update a job and save it, I cannot update many jobs then save them later

**Comment by EL-Gazoly** [comment]
We have discussed the first point before why the FE won't update because of the SSR and the function that you created to refresh the data is not working.
About the second point yes that is what it's supposed to do. do you want this functionality to be added?

**Comment by Mahmoudgalalz** [comment]
Yeah, sure and we can work on just make the changes applied just in the FE just save its state with the updated info
